streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredTimestampedKeyValueStore.java

```diff
@@ -96,7 +96,7 @@ public boolean putIfDifferentValues(final K key,
     public static class RawAndDeserializedValue<ValueType> {
```

Comment:
It seems this class is used by testing only. Could you change it from public to package-private?

Reply:
@chia7712 The code has been modified!